src/js/dataMapping/contracts/transactionTable.js

```diff
@@ -5,33 +5,38 @@
 
 const tableSearchFields = {
     columnWidths: {
-        modification: 230,
-        action_date: 200,
-        amount: 220,
-        description: 380
+        modification: 184,
```

Comment:
Those widths were based on looking at the mockup (with custom font) - reverted to original values to prevent wrapping.